docs/fx-integration.md

```diff
@@ -0,0 +1,45 @@
+# Firefox Integration
+
+As of Firefox 111, Relay has been integrated into the Firefox browser credential
+manager. This document describes the technical implementation.
+
+## Overview
+
+1. Firefox users sign into their browser with their FXA.
+2. Firefox requests an FXA access token with Relay scope for the user.
+3. Firefox sends HTTPS requests to the Relay REST API with the access token.
+
+## Firefox users sign into their browsers with their FXA
+
+Relay has to forward emails to someone's existing email address. So, to use Relay, users
+create a [Firefox Account][sumo-fxa], which requires and verifies an existing email address.
+
+TODO: Link to tech doc for Firefox/FXA integration.
```

Comment:
(suggestion, blocking) complete this TODO.

Reply:
I asked but there isn't a doc for this yet. 😢